cli/js/files.ts

```diff
@@ -22,9 +22,20 @@ import {
  *
  *       const file = Deno.openSync("/foo/bar.txt");
  */
-export function openSync(filename: string, mode: OpenMode = "r"): File {
-  const rid = sendSyncJson(dispatch.OP_OPEN, { filename, mode });
-  return new File(rid);
+export function openSync(
+  filename: string,
+  mode: OpenCapability | OpenMode = { read: true }
```

Comment:
@mhvsa should we change the parameter name, either?

Reply:
yes, forgot about that